ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -68,7 +68,14 @@ object ParallelIndexerFactory {
                   Executors.newFixedThreadPool(
                     1,
                     new ThreadFactoryBuilder().setNameFormat(s"ha-coordinator-%d").build,
-                  )
+                  ),
+                  throwable =>
+                    ContextualizedLogger
+                      .get(this.getClass)
```

Comment:
```suggestion
                      .get(getClass)
```

Reply:
Missed due to `automerge` race. Follow-up in #11707